server/events.go

```diff
@@ -219,6 +220,13 @@ type GatewayStat struct {
 	NumInbound int       `json:"inbound_connections"`
 }
 
+// JetStreamStat holds Jetstream statistics and process information.
+type JetStreamStat struct {
+	Config JetStreamConfig `json:"config,omitempty"`
+	Meta   *ClusterInfo    `json:"meta_cluster,omitempty"`
+	JetStreamStats
```

Comment:
`JetStreamStats` and `JetStreamStat` is quite confusing. Seems to me we could have used `JetStreamVarz` almost exactly as it is just adding `Meta` to it and so avoid the prolifiration of very similarly purposed structs used weirdly in different scenarios.

Reply:
moved to JetStreamVarz also set meta in Varz